deploy/cloudwash_job.yaml

```diff
@@ -66,6 +67,30 @@ objects:
                       secretKeyRef:
                         key: location
                         name: provisioning-azure-auth-secret
+            containers:
```

Comment:
This line is breaking the spec - the containers map is already defined on L24, this should be an additional item in that map.